manifests/r/rgriebl/brickstore/2022.4.1.0/rgriebl.brickstore.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rgriebl.brickstore
+PackageVersion: 2022.4.1.0
+PackageLocale: en-US
+Publisher: rgriebl

```

This must match control panel
```suggestion
Publisher: https://brickforge.de/brickstore
```